src/dotnet/commands/dotnet-list/dotnet-list-p2ps/LocalizableStrings.cs

```diff
@@ -0,0 +1,7 @@
+namespace Microsoft.DotNet.Tools.List.ProjectToProjectReferences
+{
+    internal class LocalizableStrings
+    {
+        public const string NoReferencesFound = "There are no {0} references in project {1}. ;; {0} is the type of the item being requested (project, package, p2p) and {1} is the object operated on (a project file or a solution file). ";
```

Comment:
this is copied from CommonLocalizableStrings - it seems that we did a copy too for `dotnet add` and `dotnet remove` - not sure if that's intentional - I assumed double ;; is equivalent to \n but I haven't wrote this string